lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.5); /*dim the background*/
+    vertical-align: middle;
+
+    .badges-modal {
+        color: #9b9b9b;
+        font-size: 14pt;
+        box-sizing: content-box;
+        right: 0;
+        left: 0;
+        max-width: 50%;
+        margin-right: auto;
+        margin-left: auto;
+        position: fixed;
+        top: 5em;
```

Comment:
Also, if you keep this custom value here rather than use a canned spacing value, please convert it to rems.

See our units of measurement standards here - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#units-of-measurement